rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -77,6 +77,41 @@ lv2_fs_mount_point* lv2_fs_object::get_mp(std::string_view filename)
 	return &g_mp_sys_dev_hdd0;
 }
 
+std::size_t lv2_fs_object::count_path_items(std::string_view filename)
+{
+	std::size_t count = 0;
+
+	while (true)
+	{
+		auto begin = filename.find_first_not_of('/');
+
+		if (begin == umax)
+		{
+			break;
+		}
+
+		count++;
```

Comment:
Perhaps it should handle . and .. special, as is unchanging or decreasing count.

Reply:
Yep, made similar code use this function instead because of . and ..